approved-book-list.json

```diff
@@ -1714,6 +1729,11 @@
       "content_version": "1.3.5",
       "min_code_version": "99999999.999999"
     },
+    {
+      "collection_id": "col10280",
+      "content_version": "1.5.2",
+      "min_code_version": "99999999.999999"
+    },
```

Comment:
Just FYI - You don't necessarily have to add this while the book is only for building in CORGI. We need one entry, but it doesn't really matter that it matches the version you build in CORGI.

Reply:
hello i'm so sorry i've neglected this for so long. would removing this entry resolve it?

Reply:
Actually, I wonder if this PR can just be closed 🤷‍♂️ ? I believe all of the entries are either duplicates or unnecessary.

Reply:
yes let's go ahead and do that. can you do that from your end or do i need to submit it?

Reply:
I can close it out.

Reply:
thank you!